db/migrate/20240517131641_fill_missing_file_extension_in_message_objects.rb

```diff
@@ -0,0 +1,9 @@
+class FillMissingFileExtensionInMessageObjects < ActiveRecord::Migration[7.1]
+  def up
+    MessageObject.find_each do |message_object|
+      next unless message_object.name.present?
+
+      UpdateMessageObjectNameJob.perform_later(message_object)
```

Comment:
Toto moze inak aj chvilu trvat pri tom pocte sprav co mame, aby to nedalo timeout pri deploy.

Reply:
Tak to hodim do jobu.